app/services/subcases-service.js

```diff
@@ -128,4 +128,33 @@ export default Service.extend({
     return phases;
   },
 
+  /**
+   * Finds or creates a subcase of type TYPE in your publicationFlow.
+   *
+   * @param subcaseType
+   * @param publicationFlow
+   * @param title
+   * @param shortTitle
+   * @returns {Promise<*>}
```

Comment:
subcaseType nog aanpassen
De return is geen promise, die gaat in beide gevallen een model van het type subcase terug geven.